extractor/service.go

```diff
@@ -1,20 +1,250 @@
 package extractor
 
 import (
+	"context"
+	"encoding/json"
+	"errors"
 	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	log "github.com/inconshreveable/log15"
+
+	"github.com/bblfsh/sdk/protocol"
+	"google.golang.org/grpc"
+
+	"gopkg.in/src-d/go-git.v4"                 // git.Open
+	"gopkg.in/src-d/go-git.v4/plumbing"        // Hash, Repository
+	"gopkg.in/src-d/go-git.v4/plumbing/object" // object.File
+	"gopkg.in/src-d/go-git.v4/storage"
```

Comment:
Thank you for pointing it out. 
This advice may be a great candidate for being documented style guide.

Addressed in 138d12a